src/query/query_manager.c

```diff
@@ -1147,6 +1151,7 @@ qmgr_process_query (THREAD_ENTRY * thread_p, XASL_NODE * xasl_tree, char *xasl_s
 	{
 	  goto exit_on_error;
 	}
+      query_p->includes_tde_class = xasl_p->includes_tde_class;
```

Comment:
Is there any reason to use QUERY_ENTRY? Is it impossible to use only XASL's variable?

Reply:
It is because there seems to be no way to access XASL_NODE of a query when creating a file and allocating a new page belonging to it. The some of functions which want to refer to `includes_tde_class` are followings:
`qfile_open_list(), qmgr_create_result_file(), qmgr_create_new_temp_file(), qfile_allocate_new_page(), qfile_allocate_new_ovf_page(), sort_listfile()`